test/util/validation/validate_service.go

```diff
@@ -244,6 +244,134 @@ func ValidateServicePods(ctx context.Context, k8sClient client.Client, service *
 	}).Should(gomega.Succeed())
 }
 
+// ValidateSkipModelLoaderService validates the Playground resource with llmaz.io/skip-model-loader annotation
+func ValidateSkipModelLoaderService(ctx context.Context, k8sClient client.Client, service *inferenceapi.Service) {
```

Comment:
Can we just remove the  `ValidateSkipModelLoaderService ` but make `ValidateSkipModelLoader ` part of the `ValidateService`? I just saw a lot of similar asserts here.